docs/content_management/pages/react_app_block.md

```diff
@@ -0,0 +1,118 @@
+
+Another element of React App Block is `\Ibexa\FieldTypePage\FieldType\Page\Block\Event\Listener\ReactBlock` Listener 
+which adds component and props variables.
+
+It is general for all the blocks.
+
+[[% include 'snippets/page_block_cache_clear.md' %]]
+
+## React App Block configuration
+
+React App blocks are regular [Page blocks](page_blocks.md) and can be confirgured on field definition level as any other block.
+File has exactly the same structure as regular YAML [block configuration](create_custom_page_block/#configure-block), except:
+
+- additional `component` attribute which binds Page Builder block with React App
+- `views` attribute is removed
+
+Each configured React app block has an identifier and the following settings:
+
+|Setting|Description|
+|---|---|
+| `name` | Name of the block used in the Page Builder interface. |
+| `category` | Category in the Page Builder elements menu that the block is shown in. |
+| `thumbnail` | Thumbnail used in the Page Builder elements menu. |
+| `visible` | (Optional) Toggles the block's visibility in the Page Builder elements menu. Remove the block from the layout before you publish another version of the page. |
+| `attributes` | (Optional) List of [block attributes](page_block_attributes.md). |
+
+For example:
+
+``` yaml
+ibexa_fieldtype_page:
+    react_blocks:
+        calculator:
+          name: Calculator (React)
+          category: Demo
+          thumbnail: /bundles/ibexaicons/img/all-icons.svg#date
+          component: Calculator
+          attributes:
+            a:
+              type: integer
+            b: integer
+    }
+```
+
+Each entry below `react_blocks` adds one block to the Page Builder with the defined name, category and thumbnail.
+Both name and attributes support a short syntax and a long one for specifics.
+
+`Attributes` defined without sub-keys use the key as the identifier and name, and the value as the type:
+
+```
+attributes:
+  b: integer
+```
+
+Sub-keys can be used to specify any of the usual [attributes configuration](page_block_attributes/#page-block-attributes) key:
+
+```
+attributes:
+  a:
+    name: Attribute A
+    type: string
+    options:
+      ...
+```
+
+Apps that are used as blocks are defined by default in `react/controllers` direction.
+The `assets/apps.js` file is pre-configured with this path.
+
+Apps that are registered this way must be configured and referenced in the 
+semantic configuration to be registered as blocks.
+
+Parameters passed as props must be converted so that they can be used as the configured type in the app.
+
+## Create React App block
+
+In the following example you will learn how to create `Calculator` React App block.
+
+### Configure React App Block
+
+First, create a .jsx file which describes your component.
+You can create the file in any location.
+
+In the following example, create `Calculator.jsx` file in `../../../vendor/ibexa/page-builder/components/` directory:
```

Comment:
I'm not sure if this is a good idea to write in our documentation to modify files in vendor